src/modules/involvement.js

```diff
@@ -0,0 +1,62 @@
+// https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/QaiC71NWlHq0cfCgEPcq/comments
+class Involvement {
+  constructor() {
+    this.id = '0I1gsmjKECA6HGAxhWHD';
+    this.baseURL = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${this.id}/`;
+  }
+
+  addLike = async (country) => {
+    try {
+      await fetch(`${this.baseURL}likes`, {
+        method: 'POST',
+        body: JSON.stringify({
+          item_id: country,
+        }),
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      });
+    } catch (error) {
+      throw new Error(error);
```

Comment:
Well done with using the async/await function to connect the API. The code is clear and functional.